_dashboards/index.md

```diff
@@ -11,15 +11,57 @@ redirect_from:
 
+Out-of-the-box OpenSearch Dashboards includes several tools for discovering and visualizing data, and the OpenSearch suite provides [plugins](https://opensearch.org/docs/latest/dashboards/install/plugins/) to extend and customize dashboard features and functions. 
 
-## Get started with OpenSearch Dashboards
+<img src="{{site.url}}{{site.baseurl}}/images/get-started-dashboards.png)" alt="OpenSearch Dashboards visualizations"/>
 
-1. After starting OpenSearch Dashboards, you can access it at port 5601. For example, http://localhost:5601.
-1. Log in with the default username `admin` and password `admin`.
-1. Choose **Try our sample data** and add the sample flight data.
-1. Choose **Discover** and search for a few flights.
-1. Choose **Dashboard**, **[Flights] Global Flight Dashboard**, and wait for the dashboard to load.
+# OpenSearch Dashboards use cases
+
+OpenSearch Dashboards makes it easy makes it easy for you to perform interactive log analytics, real-time application monitoring, website search, and more.  
+
+<table>
+<tbody>
+<tr>
+<td style="text-align: center; vertical-align: top;">Visualize log and trace data with interactive log analytics<img src="{{site.url}}{{site.baseurl}}/images/trace_log_correlation.gif)" alt="Log analytics"/></td>
+<td style="text-align: center; vertical-align: top;">Detect and mitigate issues faster with anomaly detection<img src="{{site.url}}{{site.baseurl}}/images/anomaly-detection.png" alt="Anomaly detection"/></td>
+</tr>
+<tr>
+<td style="text-align: center; vertical-align: top;">Use observability to monitor and issue alerts<img src="{{site.url}}{{site.baseurl}}/images/observability.png" alt="Data overview" /></td>
+<td style="text-align: center; vertical-align: top;">Unlock real-time search, monitoring, and analysis of business and operational data<img src="{{site.url}}{{site.baseurl}}/images/real-time-insights.png" alt="Real-time search, monitoring, and analysis" /></td>
+</tr>
+</tbody>
+</table> 
+
+Follow the steps below to get started creating a dashboard and exploring its elements.
+
+# Try out OpenSearch Dashboards on the playground
+
+Follow these steps to learn the basics of OpenSearch Dashboards:
+
+1.  Go to the [OpenSearch Dashboards Playground](https://playground.opensearch.org/app/home).
+2.  Choose **OpenSearch Dashboards > Dashboards > Create dashboard**. 
+3.  Explore the sample Global Flight, Web Traffic, and Revenue dashboards. 
+
+# Install OpenSearch Dashboards
+
+You can try out OpenSearch Dashboards locally after installing OpenSearch. For installation instructions, see [Install and configure OpenSearch](https://opensearch.org/docs/latest/opensearch/install/index/) and [Install and configure OpenSearch Dashboards](https://opensearch.org/docs/latest/dashboards/install/index/). 
+
+# Try out OpenSearch Dashboards locally 
```

Comment:
There's some redundancy here - I think either both headers should include "locally" or neither should, and the fact that installation is a per-requisite seems obvious enough that we can probably skip that call-out.